src/engine/loopingcontrol.h

```diff
@@ -32,14 +32,14 @@ class LoopingControl : public EngineControl {
     // process() updates the internal state of the LoopingControl to reflect the
     // correct current sample. If a loop should be taken LoopingControl returns
     // the sample that should be seeked to. Otherwise it returns currentSample.
-    double process(const double dRate,
+    virtual double process(const double dRate,
```

Comment:
process() is always a virtual, since the base class function is virtual and you cannot remove the virtual nature of a function. 
It is just a best practice to repeat the virtual keyword for documentation.